core/src/main/java/io/grpc/internal/AbstractStream.java

```diff
@@ -371,6 +375,11 @@ private void notifyIfReady() {
       boolean doNotify;
       synchronized (onReadyLock) {
         doNotify = isReady();
+        if (!doNotify) {
+          log.log(Level.FINE,
+              "Transport not ready. allocated/de:{0}/{3}, sent queued: {1}, ready thresh: {2}",
```

Comment:
This can create an immense about of logspam. It's also not at all clear what it is talking about (if you aren't looking at the line of code logs this, it is meaningless). What is this trying to achieve?

Reply:
What are you trying to achieve with this? I don't follow what it has to do with anything.

Reply:
In trying to figure out why in the interop test the onReady callback isn't being invoked I found that there is no visibility in any way into what would cause it to skip doing the notification even though the subchannel had gone into a READY state.  This was trying to provide that visibility.